database/seeding/fakeData.js

```diff
@@ -29,9 +28,13 @@ module.exports.imagesCreate = (start, stop, scale) => {
   const sp = stop * scale;
   for (let i = st; i < sp; i += 1) {
     const imageUrl = `${homeUrlBase}/home${faker.random.number({ min: 1, max: 150 })}.jpg`;
+    let text = `For Sale: $${faker.random.number({ min: 200000, max: 1500000 })}`;
```

Comment:
faker random number generation is pretty slow. you should consider writing your own random number generator function